### DISCUSSION TITLE
SD card getting wiped on Ev3

### DISCUSSION THREAD

**Discussion by techniclyrobotics** [pybricks/support#1208] [Q&A] [answered]
On one of our teams Ev3s the micro sd card keeps getting wiped. Maybe not wiped, it might just keeps not having the ev3 micropython imagine installed. Has anyone experienced this and how did you solve the problem.

**Comment by dlech** [comment]
Get a new microSD card?

Or maybe someone on your team is a hacker? 😈

**Reply by techniclyrobotics** [reply, marked as answer]
the issue has been solved it turned out to be the robot.
